### DISCUSSION TITLE
Repeatedly shifted value by one decimal place

### DISCUSSION THREAD

**Discussion by ghost** [jomjol/AI-on-the-edge-device#3439] [General] [answered]
Hi there,

I'm having a strange effect on my gas meter. About 5 ... 10x / day it reports a completely wrong value. Not just by one digit, but the whole value is either unrecognizable or shifted by one decimal place.
I spent a lot of time experimenting with the lighting. Two LED-bars arranged at an angle of ~45° (5x WS2811 each) to the glass of the meter seems to be the optimum. Unfortunately, the figures from the counter are extremely reflective. However, I think the lighting is not the cause of the problem. 

![grafik](https://github.com/user-attachments/assets/28d2b4b0-b335-430d-b8ac-59a92e52733a)

**Comment by SybexX** [comment]
What are your settings and what does the log say?
Please set the log to "Debug" for more information and then post it here after the error occurs.

**Reply by ghost** [reply]
Thanks for your time!! I've set it to 'Debug' and will post more next time this happens. 

Could you give me a hint how to post the config? As image, as txt here? What is the preferred way?

**Reply by ghost** [reply]
Here is a logfile with two successful converts and two errors: 
[DbgLog.txt](https://github.com/user-attachments/files/18140588/DbgLog.txt)

**Reply by ghost** [reply]
And here is the config-file :)

[Config.txt](https://github.com/user-attachments/files/18140620/Config.txt)

**Reply by SybexX** [reply, marked as answer]
Can you please set "Extended Resolution" to "true", if my assumption is correct this should stop the error from occurring.

**Reply by SybexX** [reply]
You can also test the customized firmware if you want (with Extended Resolution=false).

[AI-on-the-edge-device__update__v15.7.0.zip](https://github.com/user-attachments/files/18141068/AI-on-the-edge-device__update__v15.7.0.zip)

**Reply by ghost** [reply]
The "Extended Resolution" seems to have helped - no error since quite few hours! 

Many thanks!!
